src/unit.c

```diff
@@ -57,114 +50,21 @@ SEXP unit(double value, int unit)
  * (i.e., NOT unitLists or unitArithmetics)
```

Comment:
I think this comment needs an update. Maybe mention that no argument checking is performed for the sake of performance.